content/hi/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,99 @@
+---
+reviewers:
+- anubha-v-ardhan
+- divya-mohan0209
+title: कुबेरनेट्स क्या है?
+description: >
+  कुबेरनेट्स कंटेनरीकृत वर्कलोड और सेवाओं के प्रबंधन के लिए एक पोर्टेबल, एक्स्टेंसिबल, ओपन-सोर्स प्लेटफॉर्म है, जो घोषणात्मक कॉन्फ़िगरेशन और स्वचालन दोनों की सुविधा प्रदान करता है। इसका एक बड़ा, तेजी से बढ़ता हुआ पारिस्थितिकी तंत्र है। कुबेरनेट्स सेवाएँ, समर्थन और उपकरण व्यापक रूप से उपलब्ध हैं।
+content_type: concept
+weight: 10
+card:
+  name: concepts
+  weight: 10
+sitemap:
+  priority: 0.9
+---
+
+<!-- overview -->
+
+यह पृष्ठ कुबेरनेट्स का एक सिंहावलोकन है।
+
+
+<!-- body -->
+कुबेरनेट्स कंटेनरीकृत वर्कलोड और सेवाओं के प्रबंधन के लिए एक पोर्टेबल, एक्स्टेंसिबल, ओपन-सोर्स प्लेटफॉर्म है, जो घोषणात्मक कॉन्फ़िगरेशन और स्वचालन दोनों की सुविधा प्रदान करता है। Iइसका एक बड़ा, तेजी से बढ़ता हुआ पारिस्थितिकी तंत्र है। कुबेरनेट्स सेवाएँ, समर्थन और उपकरण व्यापक रूप से उपलब्ध हैं।
+
+कुबेरनेट्स नाम ग्रीक से निकला है, जिसका अर्थ है हेल्समैन या पायलट।
+K8s एक संक्षिप्त नाम के रूप में "K" और "s" के बीच आठ अक्षरों को गिनने का परिणाम है। Google ने 2014 में कुबेरनेट्स प्रोजेक्ट को ओपन-सोर्स किया।कुबेरनेट्स [गूगल के 15 से अधिक वर्षों के अनुभव](/blog/2015/04/borg-predecessor-to-kubernetes/) को समुदाय से सर्वोत्तम नस्ल के विचारों और प्रथाओं के साथ बड़े पैमाने पर उत्पादन कार्यभार को जोड़ती है।
+
+## समय पर वापस जाना
+
+आइए एक नज़र डालते हैं कि कुबेरनेट्स समय पर वापस जाने से इतना उपयोगी क्यों है।
```

Comment:
similar to above comment we can reorganize the sentence as :
`आइए समय में वापस जाकर एक नज़र डालते हैं कि कुबेरनेट्स इतना उपयोगी क्यों है। `